docs/roadmap.rst

```diff
@@ -0,0 +1,88 @@
+Roadmap
+=======
+
+Process
+-------
+
+Read the Docs has adopted the following workflow with regards to how we
+prioritize our development efforts and where the core development team focuses
+its time.
+
+Triaging issues
+~~~~~~~~~~~~~~~
+
+Much of this is already covered in our guide on :doc:`contribute`, however to
+summarize the important pieces:
+
+* New issues coming in will be triaged, but won't yet be considered part of our
+  roadmap.
+* If the issue is a valid bug, it will be assigned the ``Accepted`` label and
+  will be prioritized, likely on an upcoming point release.
+* If the issues is a feature or improvement, the issue might go through a design
+  decision phase before being accepted and assigned to a milestone. This is a
+  good time to discuss how to address the problem technically. Skipping this
+  phase might result in your PR being blocked, sent back to design decision, or
+  perhaps even discarded. It's best to be active here before submitting a PR.
+* The core team will only work on accepted issues, and will only give PR review
```

Comment:
This looks like it discourages drive-by PRs for small items. Should we add a note that if a PR is small enough, you're welcome to file it - we just may not get to it as soon as if you had gone through an issue process?

Reply:
Makes sense to me.